db/seeds.rb

```diff
@@ -35,25 +35,26 @@ def create_statuses
   end
 
   def create_items
-    Item.create!(title: "Sean's Footlong", description: "A hotdog with ketchup", price: 200)
-    Item.create!(title: "Yellow Dog", description: "A hotdog with mustard", price: 250)
+    Item.create!(title: "Turing Assessment Week", description: "A miserable dog seasoned with the tears of students", price: 200, image: File.open('app/assets/images/sad_dog.jpg'))
+    Item.create!(title: "Yellow Dog", description: "A hotdog with mustard", price: 250, image: File.open('app/assets/images/hotdog_costume.jpg'))
     Item.create!(title: "Sunset Dog", description: "A hotdog with mustard and ketchup", price: 400)
-    Item.create!(title: "Rainbow Dog", description: "A hotdog with relish, ketchup, relish, and mustard", price: 450)
-    Item.create!(title: "Salty Dog", description: "A hotdog on a pretzel bun", price: 600)
+    Item.create!(title: "Rainbow Dog", description: "A hotdog with relish, ketchup, relish, and mustard", price: 450, image: File.open('app/assets/images/everything_dog.jpg'))
+    Item.create!(title: "Salty Dog", description: "A hotdog on a pretzel bun", price: 600, image: File.open('app/assets/images/pretzel_dog.jpg'))
     Item.create!(title: "Wet Dog Smell", description: "A hotdog where the bun has been soaked in vodka", price: 1000)
-    Item.create!(title: "Standard Dog", description: "A hotdog", price: 100)
+    Item.create!(title: "Bahn Mi Dog", description: "A hotdog with picked vegetables", price: 800, image: File.open('app/assets/images/bahn_mi.jpg'))
     Item.create!(title: "Raining Cats and Dog", description: "A hotdog with a side of rain water", price: 500)
-    Item.create!(title: "Banana Split Dog", description: "A banana on a hot dog bun with whipped cream", price: 700)
-    Item.create!(title: "Sundae", description: "A banana on a hot dog bun with nuts and whipped cream", price: 800)
-    Item.create!(title: "Weiner Dog", description: "A weinerschitzel with a side of sauerkraut", price: 550)
-    Item.create!(title: "Veggie Dog", description: "A lettuce wrap with a veggie hotdog", price: 450)
+    Item.create!(title: "Banana Split Dog", description: "A banana on a hot dog bun with whipped cream", price: 700, image: File.open('app/assets/images/banana.jpeg'))
+    Item.create!(title: "Sundae", description: "A banana on a hot dog bun with nuts and whipped cream", price: 800, image: File.open('app/assets/images/sundae.jpeg'))
+    Item.create!(title: "Weiner Dog", description: "A weinerschitzel with a side of sauerkraut", price: 550, image: File.open('app/assets/images/nyc_hot_dog.jpg'))
```

Comment:
Line is too long. [163/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.